package.json

```diff
@@ -12,6 +12,7 @@
   },
   "private": true,
   "dependencies": {
+    "@angular-material-extensions/select-country": "^6.0.1",
```

Comment:
Is that needed?

Reply:
done: npm uninstall select-country

Reply:
Package is called @angular-material-extensions/select-country ;)